tests/nnstreamer_filter_tensorflow/runTest.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+##
+## @file runTest.sh
+## @author HyoungJoo Ahn <hello.ahn@samsung.com>
+## @date Dec 17 2018
+## @brief SSAT Test Cases for NNStreamer
+##
+
+if [[ "$SSATAPILOADED" != "1" ]];then
+	SILENT=0
+	INDEPENDENT=1
+	search="ssat-api.sh"
+	source $search
+	printf "${Blue}Independent Mode${NC}
+"
+fi
+
+# This is compatible with SSAT (https://github.com/myungjoo/SSAT)
+testInit $1
+
+PATH_TO_SO="$(rpm --eval '%{buildroot}%{_libdir}')/libtensor_filter_tfcore.so"
+echo $PATH_TO_SO
+OUTPUT="$(ldconfig -l $PATH_TO_SO 2>&1)"
```

Comment:
Checking with full path, hardcoded for build script is not good.

Reply:
you mean this part `/libtensor_filter_tfcore.so`?

Reply:
Other people may need to run SSAT with installed nnstreamer pacakges, not within gbs or pdebuild.

Reply:
> you mean this part /libtensor_filter_tfcore.so?

Yes.

Reply:
Hints: https://serverfault.com/questions/54736/how-to-check-if-a-library-is-installed

Reply:
> Other people may need to run SSAT with installed nnstreamer pacakges, not within gbs or pdebuild.

you're right.. I've been concentrated on only CI-pass too much. I need to concern about it.

> Hints: https://serverfault.com/questions/54736/how-to-check-if-a-library-is-installed

I already tried this way, but I can't get the result with `ldconfig -p` in my local environment.
Furthermore, `ldconfig -v | grep SOMETHING` is not working, especially `grep` is not working.